pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -187,7 +188,9 @@ func (w *Watcher) traversePluginDir(dir string) error {
 				return fmt.Errorf("failed to watch %s, err: %v", path, err)
 			}
 		case mode&os.ModeSocket != 0:
+            w.wg.Add(1)
```

Comment:
since this waitgroup is waited for somewhere else, it is not clear what why we are blocking on this following gofunc and *where*. Could you add a comment for clarity

Reply:
@davidz627 this waitGroup is only used in `Stop()` function, i believe the only caller of this `Stop()` is `Start()` when traversePlugin failed.
no one should block on this waitGroup though, since it suppose to >0 when pluginWatcher is alive.

Reply:
Added a comment